java/src/main/scala/prometheus4cats/javasimpleclient/internal/MetricCollectionProcessor.scala

```diff
@@ -209,84 +211,59 @@ class MetricCollectionProcessor[F[_]: Async: Logger] private (
         }
       }
 
-      val (_, duplicates, registryDuplicates, samples) = List(
+      val (_, registryDuplicates, samples) = List(
         collectMetrics(values.counters, counterToSample),
         collectMetrics(values.gauges, gaugeToSample),
         collectMetrics(values.histograms, histogramToSample),
         collectMetrics(values.summaries, summaryToSample)
       ).foldLeft(
         (
-          Set.empty[(Option[Metric.Prefix], String)],
           Set.empty[(Option[Metric.Prefix], String)],
           Set.empty[(Option[Metric.Prefix], String)],
           ListBuffer.empty[MetricFamilySamples]
         )
-      ) { case ((names, duplicates, registryDuplicates, samples), f) =>
-        f(names, duplicates, registryDuplicates, samples)
+      ) { case ((names, registryDuplicates, samples), f) =>
+        f(names, registryDuplicates, samples)
       }
 
-      if (duplicates.isEmpty && registryDuplicates.isEmpty) Applicative[F].pure(samples.toList)
+      if (registryDuplicates.isEmpty) Applicative[F].pure(samples.toList)
       else {
-        lazy val common =
-          "This is due to an implementation detail in the Java Prometheus library that backs the MetricsRegistry you are using,\n" +
-            "which prevents metrics with the same name and different label sets from being added to the registry, despite this being allowed by Prometheus.\n" +
-            "See here for more details: https://github.com/prometheus/client_java/issues/696\n" +
-            "These metrics will likely have been added by a library, so you may have to log an issue or raise a pull request to rectify. A stack trace has been included to help with this.\n"
-
-        lazy val duplicatesInCollection =
-          s"The following metrics with the same name and different labels or type have been discovered in a metric collection callback: '${duplicates
-              .mkString_(",")}'.\n" + common +
-            "Because of this, only the first metric will be taken and the others ignored - this will not prevent your application from functioning.\n\n" +
-            "To mitigate this you can do the following to the metric collection:\n" +
-            "  - modify the metric name to disambiguate metric types and/or label sets\n" +
-            "  - align labels between the two metrics if they are of the same type, so that they both have the same keys,\n" +
-            "    but provide empty values for the labels which are not present on each metric"
-
         lazy val duplicatesInRegistry =
-          s"The following metrics with the same name and different labels or type are already registered in the metrics registry and therefore have been excluded from the provided collection: '${registryDuplicates
-              .mkString_(",")}'" + common +
+          s"The following metrics or callbacks with the same name are already registered in the metrics registry and therefore have been excluded from the provided collection: '${registryDuplicates
```

Comment:
I'm wonder whether we need a latch on this so as not to spam peoples' logs. Maybe only logging if the known duplicates changes? The problem is that it's _another_ ref to inspect when rendering `/metrics`

Reply:
Haha yeah, Honestly, we have so many refs floating around that I wouldn't worry too much about another